### DISCUSSION TITLE
Replace the expression "powerful as hell" on the main page with something else

### DISCUSSION THREAD

**Discussion by costa100** [alpinejs/alpine#3471] [4. General]
Hi - this might seem a peculiar request, but could you please replace the expression "powerful as hell" on your main page with some other key phrasing?

Here is a suggestion:
Simple.
Lightweight.
Powerful.
Made with love.

Or you can switch the order:

Powerful.
Simple.
Lightweight.
Made with love.

Imo, you don't need to add "as hell". I know it is tempting to say it to accentuate the empowerment it offers to the developer while being so simple, but the suffix "as hell" has negative connotations. "hell" is not the most powerful thing, plus it suggests betrayal.  

Thanks :-)

**Comment by ekwoka** [comment]
> the suffix "as hell" has negative connotations

I'm not so sure this is true. "he's strong as hell". It's neither positive or negative, just a casual way to amplify the adjective like `very`.

Visually it looks nice with the length of the lines, and it has a nice rule of threes where the third item breaks the pattern (not just an adjective).

Maybe other ones are better?

`Powerful as balls`
`Powerful as heck`
`Powerful as fuck`

**Reply by costa100** [reply]
> I'm not so sure this is true

You might be right. Certain expressions are old and contain the word "hell" to emphasize a certain attribute. 
You could add "Made with passion", for instance.
English is not my first language, but seeing "Powerful as hell" feels off. It is up to you... It was a suggestion. 

> Maybe other ones are better?

😆 

Cheers

**Reply by joshhanley** [reply]
@costa100 thanks for the suggestion. I can certainly understand how it may seem off. To me though, where I am from (native English speaker) "... as hell" is pretty idiomatic and is defined as "used to make a statement more forceful" https://www.merriam-webster.com/dictionary/as%20hell

**Comment by knpwrs** [comment]
Hell isn't even powerful.

> Matthew 16:18 KJV
> And I say also unto thee, That thou art Peter, and upon this rock I will build my church; and the gates of hell shall not prevail against it.

Seems to me that `Powerful` on its own would ring better: single word, single word, single word.

**Reply by knpwrs** [reply]
How about:

- Simple.
- Lightweight.
- Powerful.
- Pick any three.

I've seen this pattern elsewhere.
